packages/webapp/pages/squads/[handle].tsx

```diff
@@ -0,0 +1,138 @@
+import {
```

Comment:
I have duplicated the source page for this. Not sure how the feed will be loaded

Reply:
We have created a new task for this